scripts/watchdog

```diff
@@ -84,7 +86,9 @@ function enable_watchdog()
         fi
     elif isMLNX; then
         debug "Calling MLNX WD enable"
-        # call watchdog api for mlnx
+        if [[ -x /usr/bin/hw-management-wd.sh  ]]; then
+            /usr/bin/hw-management-wd.sh start
```

Comment:
Stepan, can you please change this to pass the 3 minutes watchdog timer setting as you mentioned in https://github.com/Azure/sonic-buildimage/pull/4452.

hw-management-wd.sh start 180

Reply:
I will close my PR on sonic-buildimage